data/cache/cache_key.py

```diff
@@ -73,3 +73,13 @@ def for_appr_show_package(namespace, package_name, release, media_type, cache_co
     return CacheKey(
```

Comment:
nit: The appr cache keys are no longer used. Can we remove them?

Reply:
yeah I'll remove them. I also see there's a circular dep on import I'll fix that too

Reply:
You'll probably need to pass around the model_cache from the caller

Reply:
yeah too bad, also have to pass `cache_key` singleton as well